<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Al Williams</author>
 <class>KCsvEdMain</class>
 <widget class="QMainWindow" name="KCsvEdMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kcvsed</string>
  </property>
  <property name="windowIcon">
   <iconset resource="csved.qrc">
    <normaloff>:/icons/edit-4.png</normaloff>:/icons/edit-4.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroller">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>604</width>
         <height>256</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="editFrame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Navigate">
    <property name="title">
     <string>&amp;Navigate</string>
    </property>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
    <addaction name="action_First"/>
    <addaction name="action_Last"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="actionCu_t"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Insert_Row"/>
    <addaction name="action_Add_New_Row"/>
    <addaction name="action_Delete_Row"/>
    <addaction name="action_Revert"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_New_Field"/>
    <addaction name="actionEdit_Header"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Options"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Navigate"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_As"/>
   <addaction name="separator"/>
   <addaction name="action_First"/>
   <addaction name="action_Previous"/>
   <addaction name="action_Next"/>
   <addaction name="action_Last"/>
   <addaction name="separator"/>
   <addaction name="actionCu_t"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit the program</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/document-new-6.png</normaloff>:/icons/document-new-6.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create a new CSV document</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/document-open-2.png</normaloff>:/icons/document-open-2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/document-save-2.png</normaloff>:/icons/document-save-2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save current document</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/document-save-as-2.png</normaloff>:/icons/document-save-as-2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As...</string>
   </property>
   <property name="statusTip">
    <string>Save current document with new name</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/go-next-5.png</normaloff>:/icons/go-next-5.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="statusTip">
    <string>Move to next line in file (record)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/go-previous-5.png</normaloff>:/icons/go-previous-5.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="statusTip">
    <string>Move to previous line in file (record)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="action_First">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/go-first-3.png</normaloff>:/icons/go-first-3.png</iconset>
   </property>
   <property name="text">
    <string>&amp;First</string>
   </property>
   <property name="statusTip">
    <string>Move to first line in file (record)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Home</string>
   </property>
  </action>
  <action name="action_Last">
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/go-last-3.png</normaloff>:/icons/go-last-3.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Last</string>
   </property>
   <property name="statusTip">
    <string>Move to last line in file (record)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+End</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/edit-copy-3.png</normaloff>:/icons/edit-copy-3.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the current field's text to the clipboard</string>
   </property>
  </action>
  <action name="actionCu_t">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/edit-cut-6.png</normaloff>:/icons/edit-cut-6.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="statusTip">
    <string>Cut the current field's text to the clipboard</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="csved.qrc">
     <normaloff>:/icons/edit-paste-3.png</normaloff>:/icons/edit-paste-3.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste the clipboard content into the current field</string>
   </property>
  </action>
  <action name="action_Revert">
   <property name="text">
    <string>&amp;Revert Row</string>
   </property>
   <property name="statusTip">
    <string>Revert to saved copy; Warning! All changes on this row will be lost</string>
   </property>
  </action>
  <action name="action_Insert_Row">
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="statusTip">
    <string>Insert a new line (record) at the current location</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="action_Delete_Row">
   <property name="text">
    <string>&amp;Delete Row</string>
   </property>
   <property name="statusTip">
    <string>Delete current line (record)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="statusTip">
    <string>Set options</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="statusTip">
    <string>All about me</string>
   </property>
  </action>
  <action name="actionEdit_Header">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit &amp;Header</string>
   </property>
   <property name="statusTip">
    <string>Check to make header row editable</string>
   </property>
  </action>
  <action name="action_Add_New_Row">
   <property name="text">
    <string>&amp;Add New Row</string>
   </property>
   <property name="statusTip">
    <string>Add new line (record) at end of document</string>
   </property>
  </action>
  <action name="actionAdd_New_Field">
   <property name="text">
    <string>Add New Field...</string>
   </property>
   <property name="statusTip">
    <string>Add a new field to the end (adds to all records)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="csved.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>updateEditMenu()</slot>
 </slots>
</ui>
